.nvmrc

```diff
@@ -1 +1 @@
-18.17.0
```

Comment:
https://github.com/ferdium/ferdium-recipes/commit/cdafac130fd454610d1fa2cb53b061b17b64e2ce actually uses newer versions than this since recently

Reply:
damn xD I'll fix it before merging